internal/parse/parser_test.go

```diff
@@ -27,142 +29,99 @@ type M map[string]any
 
 func TestRound(t *testing.T) {
 	var tests = []struct {
-		input             string
-		expectedParsed    string
-		prepArgs          []any
-		completeArgs      []any
-		expectedCompleted string
+		input          string
+		expectedParsed string
 	}{
 		{
```

Comment:
Per chat online, can we please do these in the standard compact "cuddled" form:
```
{{
    ...
}, {
    ...
}}
```

Reply:
Ah I thought you were referring in your comment to instances such as []FullName{FullName{.... where the type was specified unnecessary in a list. Changed.

Reply:
Doesn't look like it changed. Indeed, formatting looks misaligned as well now?

Reply:
Ah, I misunderstood the comment, I've changed it and it looks a lot better now